conf/username.txt

```diff
@@ -1 +1 @@
-your-github-username-here-in-conf-file
+vido2373@colorado.edu
```

Comment:
@vido2373 no need to add the email here, you can just use your github username, vido2373